docs/sample-project.yaml

```diff
@@ -35,6 +35,11 @@ dev:
           FromPort: 4505
           ToPort: 4506
           CidrIp: 10.0.0.0/16
+      DatabaseSG:
```

Comment:
Isn't this going to create the SG on everything in the auto scaling group where as before it was created and used by the RDS.